packaging/server_upgrade_instructions.txt

```diff
@@ -18,5 +18,9 @@ following command using the service account.
 For additional information, see the help text (irodsServer -h) or
 documentation at https://docs.irods.org.
 
+If using rsyslog, please add irodsAgent to your rsyslog configuration
+to avoid missing important log messages. For detailed instructions, See
+https://docs.irods.org/5.0.0/system_overview/server_log/#rsyslog-configuration.
```

Comment:
Open issue to make version number change with each release. Or open issue to remove this in a later release - but which one? 5.1 or 6.0?

Reply:
Created #8536.